src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -869,6 +869,27 @@ public function foreignIdFor($model, $column = null)
                     : $this->foreignUuid($column ?: $model->getForeignKey());
     }
 
+    /**
+     * Creates a foreign ID column and key for the given model.
+     *
+     * @param \Illuminate\Database\Eloquent\Model|string $model
+     * @param string|null $column
+     * @param string|null $table
+     * @param string|null $keyName
+     * @return ForeignKeyDefinition
```

Comment:
FQCN please

Reply:
@zeo this isn't resolved so please update the DocBlocks. Also make sure they have the same format (two spaces before and after types for the `@param` annotation.